docker-compose.yml

```diff
@@ -375,4 +395,5 @@ services:
       - osf_node_modules_vol:/code/node_modules
       - preprints_dist_vol:/preprints
       - registries_dist_vol:/registries
+      - osfstoragecache_vol:/code/website/osfstoragecache
```

Comment:
this does not need to be here, osf merely directs wb where to store its temp files relative to the wb process